build.gradle

```diff
@@ -65,3 +65,7 @@ task functional(type: Test) {
   }
 }
 tasks.functional.dependsOn("myZip");
+
+run {
+    args System.getProperty("exec.args", "").split()
+}
```

Comment:
perhaps perform a clean prior to run so that it will be a clean build every time?

Reply:
also perhaps `run.args` would be more intuitive?

Reply:
```args``` looks good too. 

Then running it would be:
```$ ./gradlew run -Dargs="-config CSV_path.csv -output output_path/ -since 01/10/2017 -until 01/11/2017"```